test/v1/models/request/sample/disclosures/AmendTaxAvoidanceSpec.scala

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.models.request.sample.disclosures
+
+import play.api.libs.json.{JsError, JsValue, Json}
+import support.UnitSpec
+import v1.models.request.disclosures.AmendTaxAvoidance
+
+class AmendTaxAvoidanceSpec extends UnitSpec {
+
+  private val json: JsValue = Json.parse(
```

Comment:
space out json and use valid examples like the commented json in the AmendDisclosuresRequestBodySpec